pycondor/tests/test_dagman.py

```diff
@@ -293,3 +296,29 @@ def test_dagman_add_node_ignores_duplicates(tmpdir, dagman):
     dagman.add_job(job)
 
     assert dagman.nodes == [job]
+
+
+def test_dagman_build_single_file(tmpdir):
```

Comment:
This test should be expanded to ensure that all `Job` submit file attributes (e.g. `queue`, `log`, etc.) are included when `single_file=True`. Using the `pytest.mark.parametrize` decorator could be useful here.